libs/dh/market-participant/actors/feature-actors/src/lib/create/dh-actors-create-actor-modal.component.ts

```diff
@@ -81,33 +113,53 @@ import { dhCvrValidator, dhDomainValidator } from '@energinet-datahub/dh/shared/
 export class DhActorsCreateActorModalComponent {
   private _fb: NonNullableFormBuilder = inject(NonNullableFormBuilder);
   private _apollo = inject(Apollo);
+  private _destroyRef = inject(DestroyRef);
 
   private _getOrganizationsQuery$ = this._apollo.watchQuery({
     useInitialLoading: true,
     notifyOnNetworkStatusChange: true,
     query: GetOrganizationsDocument,
   });
 
+  private _getOrginationByIdQuery$ = this._apollo.watchQuery({
+    useInitialLoading: true,
+    notifyOnNetworkStatusChange: true,
+    query: GetOrganizationByIdDocument,
+  });
+
   @ViewChild(WattModalComponent)
   innerModal: WattModalComponent | undefined;
 
   organizationOptions: WattDropdownOptions = [];
+  marketRoleOptions: WattDropdownOptions = dhEnumToWattDropdownOptions(EicFunction);
   countryOptions: WattDropdownOptions = [
     { value: 'DK', displayValue: 'DK' },
     { value: 'SE', displayValue: 'SE' },
     { value: 'NO', displayValue: 'NO' },
     { value: 'FI', displayValue: 'FI' },
   ];
 
-  chooseOrganizationForm = this._fb.group({ orgId: [''] });
+  chooseOrganizationForm = this._fb.group({ orgId: ['', Validators.required] });
   newOrganizationForm = this._fb.group({
     country: ['', Validators.required],
     cvrNumber: ['', [Validators.required, dhCvrValidator()]],
     companyName: ['', Validators.required],
     domain: ['', [Validators.required, dhDomainValidator]],
   });
 
-  showCreateNewOrganization = false;
+  newActorForm = this._fb.group({
+    glnOrEicNumber: ['', [Validators.required, dhEicOrGlnValidator]],
+    name: [''],
+    marketrole: ['', Validators.required],
+    contact: this._fb.group({
+      departmentOrName: ['', Validators.required],
+      email: ['', [Validators.required, Validators.pattern(/^[A-Za-z0-9]+$/)]],
```

Comment:
suggestion: `dh-actors-edit-actor-modal.component.ts` has a slightly different pattern for e-mails. Let's consolidate the two and reuse it in both places.